golem-base/query/eval_test.go

```diff
@@ -41,6 +47,27 @@ func TestEqualExpr(t *testing.T) {
 	require.NoError(t, err)
 
 	require.Equal(t, []common.Hash{common.HexToHash("0x1")}, res)
+
+	// Query for a key with special characters
+	expr, err = query.Parse("déçevant = \"non\"")
+	require.NoError(t, err)
+
+	res, err = expr.Evaluate(ds)
+	require.NoError(t, err)
+
+	require.Equal(t, []common.Hash{common.HexToHash("0x3")}, res)
+
+	expr, err = query.Parse("بروح = \"ايوة\"")
```

Comment:
Github is having some issues with RTL here, at least in firefox. The strings are properly inside the quotes in reality, even though it looks here like they aren't.